doc/source/manual/state_snapshot_transfer.rst

```diff
@@ -45,7 +45,7 @@ Using Percona Xtrabackup
 
 The default SST method is ``xtrabackup-v2`` which uses |Percona XtraBackup|.
 This is the least blocking method that leverages `backup locks
-<http://www.percona.com/doc/percona-server/5.7/management/backup_locks.html>`_.
+<http://www.percona.com/doc/percona-server/8.0/management/backup_locks.html>`_.
 XtraBackup is run locally on the donor node.
 
 The :term:`datadir` needs to be specified in the server configuration file
```

Comment:
Should either link into DARE docs or separately discuss SST / DARE interactions.

Should also talk about our changed default in PXC 8.0 to have replication traffic encrypted by default which makes SST fail if you don't have a correct key, and solves the potential breach that existed in prior release series.